### ISSUE TITLE
High-resolution icon

### ISSUE THREAD

**Issue by nicolas-raoul** [zakkudo/gwaei#3]
Icon could look better. On HDPI display:

![gwaei-icon](https://cloud.githubusercontent.com/assets/99590/13975932/08fb3a36-f0fe-11e5-9c0c-33f26a056cd8.png)

**Comment by zakkudo** [comment]
Good point. I just updated to a high hdpi display so I think i'm going to start feeling this

**Comment by nicolas-raoul** [comment]
If you could include the SVG (or other vectorial format) to Github that woud be the most future-proof :-)

**Comment by zakkudo** [comment]
That is a good point...  especially now that I am on a high dpi display
